recipes/arcticdb/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "articdb" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # This is the last revision of `experimental/conda-forge-release`
+  # url: https://github.com/man-group/ArcticDB/archive/5c7ef369e23c5c168548888e9a8bdc2a9dbd6db1.tar.gz
+  # sha256: e130588a84ad6e271d12b28fb9f0da64638f128a6d53c02f98f0c31ffcf297d8
+  git_url:  https://github.com/man-group/ArcticDB
+  git_rev:  experimental/conda-forge-release
+
+build:
+  # We skip the build on Windows because one of the dependencies (folly)
+  # is not available for Windows on conda-forge
+  # See: https://github.com/conda-forge/folly-feedstock/pull/98
+  skip: true  # [win]
+  script: ARCTICDB_USING_CONDA=1 ARCTICDB_BUILD_CPU_COUNT=1 {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  # Compiling ArcticDB with all cores might freeze machines.
  # We build with only 1 core to prevent freezes from happening.
  script: ARCTICDB_USING_CONDA=1 CMAKE_BUILD_PARALLEL_LEVEL=1 {{ PYTHON }} -m pip install . -vv
```